app/views/claims/support/schools/remove_grant_conditions_acceptance_check.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_support_school_path(@school)) %>
+<% end %>
+
+<% content_for :page_title, t(".page_title", user_name: @school.claims_grant_conditions_accepted_by.full_name) %>
```

Comment:
To associate this with how things are rendered on the page, can we bring this up to line 1 and then call the `before_content` block please?